### PR TITLE
1 project boilerplate project initialized

### PR DESCRIPTION
**Pull request by Viktor-K**
**Test Plan**
start the app with 'yarn start'

Read more about boilerplate changes in [https://github.com/buildo/bento/pull/117#issuecomment-723660454](url)

### CODE REVIEW

**Comment by Viktor-K** [comment]
@veej should be a big impact update Typescript version to 4 ? in previous template TS version is 3.

**Comment by veej** [comment]
> @veej should be a big impact update Typescript version to 4 ? in previous template TS version is 3.

It should not have big impacts. v4 doesn't introduce any big change:
https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/

> With TypeScript 4.0, there are no major breaking changes. In fact, if you’re new to the language, now is the best time to start using it.

File: src/API/API.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ),
+    taskEither.mapLeft(() => genericError),
+    taskEither.chain((res) =>
+      pipe(
+        t.array(t.string).decode(res),
+        either.mapLeft(decodingError),
+        taskEither.fromEither
+      )
+    )
+  );
```
Comment (veej) [inline src/API/API.ts:34]: This is not needed, but you can replace it in future PRs with the real calls to the yelp API

File: src/commands/commands.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { viewToLocation } from "../model";
+
+export const doUpdateCurrentView = getDoUpdateCurrentView(viewToLocation);
+
+export const doRefreshUsername = command(
+  // doesn't perform any operation: it exists only to refresh the `randomName` query
+  () => taskEither.of<void, void>(undefined),
+  // when successful, should invalidate this query
+  { randomNames }
+);
```
Comment (veej) [inline src/commands/commands.ts:20]: same here

File: src/components/Hello/Hello.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
```
Comment (veej) [inline src/components/Hello/Hello.tsx:1]: please, make sure to remove all these files from future PRs, since they're just for demo purposes

**Review by veej — COMMENTED** [review]
LGTM 🎉

**Pull request merged**